appveyor.yml

```diff
@@ -6,6 +6,7 @@ cache:
 install:
   - ps: Install-Product node $env:nodejs_version x64
   - yarn
+  - yarn run prepare
   - git submodule sync
```

Comment:
actually this is needed because it's an older version of yarn that doesn't run prepare automatically on `yarn` like it is supposed to. I did have to bump the yarn version in circle.yml (which is why it's in this PR) to make that happen.